runtime/src/core/grain-module.js

```diff
@@ -80,6 +80,7 @@ export class GrainModule {
 }
 
 export async function readFile(path) {
+  const fs = require('fs');
```

Comment:
Oops.

Reply:
Not really your fault. This is specifically necessary since we can't require `fs` in the browser.